Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP.py

```diff
@@ -478,6 +558,86 @@ def get_alerts_command():
     demisto.results(entry)
 
 
+def upload_yara_file_command(client, args):
+
+    entry_id = args.get('entryID')
+    policy_uuid = args.get('policy_uuid')
+    ruleset_uuid = args.get('ruleset_uuid')
+
+    file_obj = demisto.getFilePath(entry_id)
+    file_path = file_obj['path']
+
+    with open(file_obj.get('path'), "rb") as file:
+        data=file.read()
+        files = {
+            'file': ('new.yara', data)
+        }
+        response = client.upload_yara_file(policy_uuid, ruleset_uuid, files)
+        #print (response2)
+        if response.status_code == 202:
+            return "Upload of Yara file succesfully."
+        else :
+            return "Upload of Yara file failed."
+
+
+def get_events_data_command(client, args):
+
+    message_id = args.get('message_id')
+
+    response = client.get_events_data(message_id)
+
+
+    result_output = dict()
+    result_output['Logs'] = response['data'][message_id]
```

Comment:
why saving to a dictionary is needed? it may be redundant.

Reply:
Hi, i make new dict to send it like output

Reply:
great